usagov/vote-gov-drupal
# Vote-418: Homepage Scaffolding

## Jira ticket

[Vote-418](https://cm-jira.usa.gov/browse/VOTE-418)

## Description

This PR will add a new layout for Landing Page

## Deployment and testing

### Pre-deploy

n/a

### Post-deploy

n/a

### QA/Test

1. pull down branch 
2. cd into the new `votegov` them dir 
3. run `npm i`, `npm run build` and `lando rebuild -y` 
4. you may also need to clear cache with `lando drush cr`
5. visit the home page and ensure that the content is there and it is full width of page as well

## Checklist for the Developer




## Checklist for the Peer Reviewers
